app/javascript/listings/listings.jsx

```diff
@@ -302,25 +302,24 @@ export class Listings extends Component {
         />
         {shouldRenderModal && (
           <Fragment>
-            <div className="crayons-modal">
-              <Modal
-                currentUserId={currentUserId}
-                onAddTag={this.addTag}
-                onChangeDraftingMessage={this.handleDraftingMessage}
-                onClick={this.handleCloseModal}
-                onChangeCategory={this.selectCategory}
-                onOpenModal={this.handleOpenModal}
-                onSubmit={this.handleSubmitMessage}
-                listing={openedListing}
-                message={message}
-              />
-              <ModalBackground onClick={this.handleCloseModal} />
-            </div>
-            <KeyboardShortcuts
-              shortcuts={{
-                Escape: this.handleCloseModal,
-              }}
-            />
+            <FocusTrap
+              onDeactivate={() => this.handleCloseModal('close-modal')}
+            >
+              <div className="crayons-modal">
+                <Modal
```

Comment:
Out of scope for this PR, but we should really just use the crayons modal for this eventually. I have it on a list of to dos.

Reply:
100% - I have a wee list of additional issues we might want later, including this 🙂

I would have ported across to the 'normal' modal here but it looked like it needed some closer attention to be able to maintain the current styles etc